activities/MindMath.activity/css/activity.css

```diff
@@ -327,13 +327,15 @@ body {
   font-family: system-ui;
   font-size: 11px;
   width: 245px;
+  
```

Comment:
Do not change indentation

Reply:
Yes got it. I feel this issue persists in each and every activity, so I feel this can be solved.